.gitattributes

```diff
@@ -1,3 +1,5 @@
+* text=auto eol=lf
```

Comment:
This ensures files are checked-out with linux file endings on Windows.That is usually not the recommendation for windows. But this is the best option given how many input-output files in the tests system that are manipulated by posix tools.